perun-core/src/main/java/cz/metacentrum/perun/core/implApi/UsersManagerImplApi.java

```diff
@@ -397,6 +397,15 @@
 	 */
 	void checkUserExtSourceExists(PerunSession perunSession, UserExtSource userExtSource) throws InternalErrorException, UserExtSourceNotExistsException;
 
+	/**
+	 * Check if userExtSource exists in underlaying data source by its ID.
+	 *
+	 * @param perunSession
+	 * @param id
+	 * @throws InternalErrorException
```

Comment:
There should be information about returned value of this method. But check methods should not return any value.